addons/product_images/wizard/product_fetch_image_wizard.py

```diff
@@ -175,8 +175,9 @@ def _get_products_to_process(self, limit=10000):
             # p.image_fetch_pending needed for self.products_to_process's records that might already
             # have been processed but not yet removed from the list when called from
             # action_fetch_image.
-            lambda p: not p.image_1920 and p.barcode and p.image_fetch_pending
-        )[:limit]  # Apply the limit after the filter with self.products_to_process for more results
+            lambda p: not p.image_1920 and p.barcode and p.image_fetch_pending,
+            limit=1,  # Apply the limit after the filter with self.products_to_process for more results
```

Comment:
I only saw this error (for the Sales part)
```suggestion
            limit=limit,  # Apply the limit after the filter with self.products_to_process for more results
```
Interesting feature by the way, quite easy to use and to understand